Gopkg.lock

```diff
@@ -122,11 +122,12 @@
   name = "github.com/oracle/oci-go-sdk"
   packages = [
     "common",
+    "common/auth",
     "core",
     "identity"
   ]
-  revision = "8090aeb474ad994446b2db294725e3d6c0d95327"
-  version = "v1.0.0"
+  revision = "483be76c8d79ca38dd2e7e7d1ee0047d5cc2c66b"
+  version = "v1.6.0"
```

Comment:
Not sure how it's happening but doesn't look in-sync with Gopkg.toml.

Reply:
I think it just automatically upgrades it to version 1.6.0. I'll bump the version in the toml file to 1.6.0 to match it